src/parser/jobs/drk/index.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
-
+import {Trans} from '@lingui/react'
 import CONTRIBUTORS, {ROLES} from 'data/CONTRIBUTORS'
 import {Meta} from 'parser/core/Meta'
 
 export default new Meta({
 	modules: () => import('./modules' /* webpackChunkName: "jobs-drk" */),
 
 	Description: () => <>
-		<p>This analyzer focuses on blood and mana usage, and then further explores common resource generation problems such as Blood Weapon usage and dropping GCDs.</p>
-		<p>Most of the data in this evaluation tool is simulated, and edge cases from FFLogs can cause weird results.  If something doesn't look right, please report it in the discord.</p>
+		<Trans id="drk.about.description">
```

Comment:
nit; doesn't need the fragment (`<>...</>`) around it any more. Could use `TransMarkdown` for this cut the tags out of the translation string.